internal/config/config.go

```diff
@@ -31,6 +31,9 @@ type Config struct {
 	// CollectInterval specifies how often otel-exporter will collect metrics from Firebolt. It will also define
 	// a discretion step of reported metrics.
 	CollectInterval time.Duration `env:"FIREBOLT_OTEL_EXPORTER_COLLECT_INTERVAL,default=30s"`
+
+	// Database for which table metrics will be fetched
+	Database string `env:"FIREBOLT_OTEL_EXPORTER_DATABASE"`
```

Comment:
My preference would also be to explicitly configure the engines, and not fetch the engine list via https://github.com/alexanderchiu/otel-exporter/blob/97465b4d2c2b9eb685df64b825e6bca5b2252b38/internal/fetcher/fetcher.go#L47